src/test/java/com/synerzip/expenseCalculation/ExpenseControllerTest.java

```diff
@@ -65,13 +72,23 @@ public void testAddExpense() throws Exception {
 		jObject.put("date", "2013-09-18");
 		jsonExpense = jObject.toString();
 
-		BDDMockito.given(expenseService.addExpense(65, expense)).willReturn(anyExpense);
+		BDDMockito.given(expenseService.addExpense(expense)).willReturn(anyExpense);
 
-		mockmvc.perform(post("/expenses/65")
+		mockmvc.perform(post("/users/expenses")
 						.contentType(MediaType.APPLICATION_JSON)
 						.content(jsonExpense))
-			   .andExpect(status().isOk());
+			   .andExpect(status().isOk())
```

Comment:
you are supposed to testing whether the resource is created or not.
so your expected status should be isCreated.